### PR TITLE
Refactor character module and add docstring

### LINKED ISSUE #3: Reduce bloated text before embeds
Right now there are a lot of extra messages in front of the result embed. While potentially useful, they clog up the space too much. Those texts need to be reworded/moved to another location.

### LINKED ISSUE #4: Create help command/page
Currently documentation for how to use the bot is very lacking. A help command or help page is needed.

### PR DESCRIPTION
**Pull request by nnguyen259**
This will close #4 and #3

### CODE REVIEW

**Review by iyZAIRI — DISMISSED** [review]

**Review by elee-github — DISMISSED** [review]

**Comment by nnguyen259** [comment]
Wait for #10 before merge

- [x] Merge #10 first

File: cogs/character.py
Comment (iyZAIRI) [inline cogs/character.py]: basis

File: cogs/character.py
Comment (iyZAIRI) [inline cogs/character.py]: replaces instead of replace

File: cogs/character.py
Comment (iyZAIRI) [inline cogs/character.py]: versions

File: cogs/character.py
Comment (iyZAIRI) [inline cogs/character.py]: List

File: cogs/prefix.py
Comment (iyZAIRI) [inline cogs/prefix.py]: Basis

**Review by iyZAIRI — DISMISSED** [review]

**Review by elee-github — DISMISSED** [review]

**Pull request merged**
